app/assets/src/components/common/UserContext.jsx

```diff
@@ -3,6 +3,8 @@ import React from "react";
 export const UserContext = React.createContext({
   admin: false,
   allowedFeatures: new Set(),
+  appConfig: {},
```

Comment:
this is still abusing the concept of `UserContext` because it is does not depend on the user... any comment?

Reply:
Yeah, Tiago and I discussed this offline. We're going to rename `UserContext` to `AppContext` and possibly rename variables like `admin` to be more clear with that renaming (for example, `admin` might become `isUserAdmin`). Didn't want to do in this PR.